apps/website/src/components/layout/Layout.tsx

```diff
@@ -25,10 +24,10 @@ export const ptSerif = PT_Serif({
 export const fonts = `${ptSans.variable} ${ptSerif.variable} font-sans`;
 
 const Layout: React.FC<LayoutProps> = ({ children }) => {
-  const router = useRouter();
-  if (router.pathname.endsWith("/attachment")) {
-    return <>{children}</>;
-  }
+  // Add fonts to body for portals that do not attach to #app
+  useEffect(() => {
+    document.body.classList.add(...fonts.split(" "));
+  }, []);
```

Comment:
Could you update the consent overlay to not set these now, as we're setting them here globally?

Reply:
Oh I forgot we already have that dialog. Sure. Or I should we use your way for the ModalDialog? Not sure whats better

Reply:
Not sure I have much of a preference either way -- this is probably easier so we don't have to think about it for anything else in the future that is portal'd